src/renderer/components/ft-profile-channel-list/ft-profile-channel-list.js

```diff
@@ -41,7 +41,11 @@ export default defineComponent({
   },
   computed: {
     backendPreference: function () {
-      return this.$store.getters.getBackendPreference
+      let preference = this.$store.getters.getBackendPreference
+      if (preference === 'piped') {
```

Comment:
Is there not way to eliminate this duplicate code?

Reply:
Not that I'm aware of but most of this will be removed in follow up PRs as I start to implement piped elsewhere